lib/modern_searchlogic/column_conditions.rb

```diff
@@ -72,6 +72,19 @@ def searchlogic_arel_alias(searchlogic_suffix, arel_method, options = {})
         end
       end
 
+      def searchlogic_active_record_alias(searchlogic_suffix, options = {}, &block)
+        searchlogic_suffix_condition "_#{searchlogic_suffix}" do |column_name, *args|
+          raise ArgumentError, "wrong number of arguments (0 for >= 1)" if args.empty?
+          raise ArgumentError, "unsupported searchlogic suffix #{searchlogic_suffix} passed" unless [:in, :not_in].include?(searchlogic_suffix)
+
+          if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
`unscoped do`

Reply:
The above code changed somewhat on Friday so I don't know if the above comment is still valid with the latest changes.

Reply:
yep this was resolved